src/fsharp/service/SemanticClassificationKey.fsi

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.SourceCodeServices
+
+open System
+
+open FSharp.Compiler.Text
+
+/// Provides a read only view to iterate over the contents of the semantic classification key store.
+[<Sealed>]
+type SemanticClassificationKeyStoreView =
```

Comment:
Yes, I believe this is the right thing to make public. :)

Reply:
The name `SemanticClassificationKeyStoreView` might be a little too specific at it relates to the implementation details of the `KeyStore` itself; maybe `SemanticClassificationView` is enough.

Reply:
Sure, sounds reasonable :)